src/crsAdapter/TomaAdapter.js

```diff
@@ -651,11 +741,11 @@ class TomaAdapter {
             service.dropOffLocation,
         ].join('');
 
-        xml['From.' + lineNumber] = pickUpDate.isValid() ? pickUpDate.format(CONFIG.crs.dateFormat) : service.pickUpDate;
-        xml['To.' + lineNumber] = dropOffDate.isValid() ? dropOffDate.format(CONFIG.crs.dateFormat) : service.dropOffDate;
         xml['Accommodation' + lineNumber] = pickUpTime.isValid() ? pickUpTime.format(CONFIG.crs.timeFormat) : service.pickUpTime;
         xml['Count.' + lineNumber] = service.milesIncludedPerDay;
         xml['Occupancy.' + lineNumber] = service.milesPackagesIncluded;
+        xml['From.' + lineNumber] = pickUpDate.isValid() ? pickUpDate.format(CONFIG.crs.dateFormat) : service.pickUpDate;
```

Comment:
Totally doesn't matter at all, but I happened to wonder why this was rearranged?

Reply:
I want that this order fits the order of the CRS UI fields. Maybe it makes the orientation in the code and future implementations easier.